### ISSUE TITLE
Lists do not look like lists

### ISSUE THREAD

**Issue by yatil** [w3c/wbs-design#71]
In the new CSS, lists have no bullets and are not indented. This makes certain questions hard to read. Please reinstate the default browser list rendering in questions.

(Inline styles are stripped from questions, so there is no way to properly format lists at the moment.)

**Comment by lcarcone** [comment]
I'm working to reinstate the default style for lists in the description of the question (ul are also used in the widgets so we have to handle both cases).
Note that the ordered lists are rendered with the default style in the current version, only unordered lists are concerned

**Comment by yatil** [comment]
Thanks, @lcarcone!

**Comment by lcarcone** [comment]
The fix has been installed in wbs code, that should be ok now.
Thanks
